Formula/confluent-hub-client.rb

```diff
@@ -0,0 +1,13 @@
+class ConfluentHubClient < Formula
+  desc "Confluent Hub Client"
+  homepage "https://confluent.io"
+  url "https://s3-us-west-2.amazonaws.com/confluent-hub-client/confluent-hub-client-5.1.0-beta180619025141-package.tar.gz"
+  version "5.1.0-beta180619025141"
```

Comment:
We should not be releasing 5.1.0 betas. Let's use 5.0.0 beta instead. (This is probably because the PR was against master.)

Reply:
Thanks, updated to use `5.0.0-beta201806200000` version.